{
  "debianbug": 490217,
  "description": "PyDNS (aka python-dns) before 2.3.1-5 in Debian GNU/Linux does not use random source ports for DNS requests and does not use random transaction IDs for DNS retries, which makes it easier for remote attackers to spoof DNS responses, a different vulnerability than CVE-2008-1447.  NOTE: this vulnerability exists because of an incomplete fix for CVE-2008-4099.",
  "releases": {
    "bullseye": {
      "fixed_version": "2.3.1-5",
      "repositories": {
        "bullseye": "2.3.6-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "2.3.1-5",
      "repositories": {
        "buster": "2.3.6-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "2.3.1-5",
      "repositories": {
        "jessie": "2.3.6-3"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "2.3.1-5",
      "repositories": {
        "sid": "2.3.6-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "2.3.1-5",
      "repositories": {
        "stretch": "2.3.6-3"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}